shell/common/common_benchmarks.sh

```diff
@@ -17,6 +17,14 @@ get_bench_suites() {
   echo -e "${defined_benchs:0:(-1)}" #remove trailing space
 }
 
+validate_bench_data_size() {
+  if [[ "$1" =~ ^[0-9]+(\.[0-9]+)?([MGT]B)?$ ]]; then
+    return 0
+  else
+    return 1
+  fi
+}
```

Comment:
No need to implement your own return codes, just use the one that will be returned automatically:

```
validate_bench_data_size() {
    [[ $1 =~ ^[0-9]+(\.[0-9]+)?([MGT]B)?$ ]]
}
```